test/tests/def_blocks.html

```diff
@@ -24,5 +25,6 @@
 <blockquote>
   <p>foo
 bar
+[1]: foo
```

Comment:
There's a problem in https://github.com/KostyaTretyak/marked/blob/98ac7a43958014832788b9d01580f394c84bfb27/test/tests/def_blocks.text :
The `[1]` at line 2 is interpreted as a shortcut link to link definition at line 20, so it can be confusing. However, I think it (line 2) should be parsed as a link def itself, because it follows the correct syntax, event if it is inside a blockquote, because it is allowed by the [commonmark spec](http://spec.commonmark.org/0.28/#example-181).

Reply:
As we now know, this edit was correct because we have rule [A link reference definition cannot interrupt a paragraph.](http://spec.commonmark.org/0.28/#example-178).

Reply:
There is no broad consensus on this one:
http://johnmacfarlane.net/babelmark2/?normalize=1&text=%3E+foo%0A%3E+bar%0A%5B1%5D%3A+foo%0A%3E+bar

However, commonmark.js parses it as you wrote.